example_school_library_decorator/book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
```

Comment:
- [ ] A book should have many rentals. Therefore in the book class you are required to implement a method for adding rentals. Kindly refer to the screenshot  below from the requirements: 
![image](https://user-images.githubusercontent.com/68167810/132856809-6ae53424-c99e-44a8-afb0-9f06194d7663.png)
[Here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship`) is a link to look at about this lesson